docs/data/joy/components/circular-progress/circular-progress.md

```diff
@@ -68,6 +68,27 @@ The `determinate` prop lets you indicate a specified wait time.
 
 {{"demo": "CircularProgressDeterminate.js"}}
 
+### Determinate Count Up
+
+This method uses an external `use-count-up` package, to animate counting up by providing `start, end and duration` values.
+
+<codeblock storageKey="package-manager">
+```bash npm
+npm install use-count-up
+```
+
+```bash yarn
+yarn add use-count-up
+```
+
+```bash pnpm
+pnpm add use-count-up
+```
+
+</codeblock>
```

Comment:
```suggestion

```

I'd cut the installation block here and just mention the package link, as suggested below.